### DISCUSSION TITLE
yourls.org cookie Page Not Found

### DISCUSSION THREAD

**Discussion by wyfang** [YOURLS/YOURLS#3748] [Q&A] [answered]
https://yourls.org/cookie
![WiFi- 2024-04-16 at 14 27 31@2x](https://github.com/YOURLS/YOURLS/assets/17900420/73eee093-b93f-4171-a677-1db5134ca883)

**Comment by dgw** [comment, marked as answer]
Same cause as #3747: part of the website infrastructure is down.

~~We are investigating.~~ It has been fixed! Thanks for letting us know!

**Reply by dgw** [reply]
The issue has been fixed today. Thanks again for notifying us of the problem!
